packages/block-editor/src/components/list-view/use-list-view-scroll-into-view.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { getScrollContainer } from '@wordpress/dom';
+import { useLayoutEffect } from '@wordpress/element';
+
+export default function useListViewScrollIntoView( {
+	isSelected,
+	selectedClientIds,
+	rowItemRef,
+} ) {
+	const isSingleSelection = selectedClientIds?.length === 1;
```

Comment:
Nit: Is `selectedClientIds` required in this hook? If it is then we probably need the nullish check here.

Reply:
What did you have in mind for this one? Would it be something like the following:

```
const isSingleSelection = selectedClientIds ?? selectedClientIds?.length === 1;
```

If so, I'm not sure if it's needed, because it doesn't appear to cause any errors if `selectedClientIds` isn't passed in, since the destructuring assignment for `selectedClientIds` means that the optional chaining doesn't appear to throw an error because is `selectedClientIds` is defined as undefined, rather than being truly undefined... I think 😅

For example, I tried the following in the console:

<img width="491" alt="image" src="https://user-images.githubusercontent.com/14988353/218363304-18916638-6679-40b0-8787-dce5e4f3df95.png">

Was there another case you were thinking of, though, where adding a nullish check would improve things? Happy to keep tweaking this!

Also thanks for your comment, TIL that optional chaining doesn't perform a check for us to make sure that the variable is defined before attempting to access the property, so I'm very happy learn more here, too 🙂

Reply:
Oh, oops! I mean optional chaining, not nullish operator 😅 .

```suggestion
	const isSingleSelection = selectedClientIds.length === 1;
```

Sorry for the misunderstanding!

Reply:
Ah, excellent, thanks for clarifying! Yes, `selectedClientIds` should always be both required and an array, from looking at [the selector](https://github.com/WordPress/gutenberg/blob/f50ab9815d6b8ae00781a7db8c31e893af0ac19f/packages/block-editor/src/store/selectors.js#L710), it'll always return an array, even if it's empty, so this looks safe to me 👍